orderer/common/cluster/deliver_test.go

```diff
@@ -180,9 +180,23 @@ func (ds *deliverServer) Deliver(stream orderer.AtomicBroadcast_DeliverServer) e
 	if err != nil {
 		panic(err)
 	}
+
+	// FAB-16233 This is meant to mitigate timeouts when
+	// seekAssertions does not receive a value
+	timeoutC := make(chan bool, 1)
```

Comment:
We can do better. In addition a bunch of go routines, it’s a ton of boilerplate that is unnecessary; just use a Timer.

timer := time.NewTimer(3*time.Minute)
defer timer.Stop()

The select can then use timer.C.